dev/com.ibm.ws.logging.osgi/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -14,76 +14,73 @@
 #NLS_ENCODING=UNICODE
 #NLS_MESSAGEFORMAT_NONE
 #
-
 logging.properties=Logging
 logging.properties.desc=Controls the capture and output of log and trace messages.
 
 #
 # File size / max files
 #
 max.file.size=Maximum log file size
-max.file.size.desc=Maximum size of a log file, in megabytes, before being rolled over; a value of 0 means no limit.
+max.file.size.desc=The maximum size (in MB) that a log file can reach before it is rolled. The Liberty runtime does only size-based log rolling. To disable this attribute, set the value to 0. The maximum file size is approximate. By default, the value is 20. Note: maxFileSize does not apply to the console.logfile.
 
 max.files=Maximum log files
-max.files.desc=Maximum number of log files that will be kept, before the oldest file is removed; a value of 0 means no limit.
+max.files.desc=Maximum number of log files that will be kept, before the oldest file is removed; a value of 0 means no limit. If an enforced maximum file size exists, this setting is used to determine how many of each of the logs files are kept. This setting also applies to the number of exception logs that summarize exceptions that occurred on any particular day. So if this number is 10, you might have 10 message logs, 10 trace logs, and 10 exception summaries in the ffdc/directory. By default, the value is 2. Note: maxFiles does not apply to the console.log file.
 #
 # File name
 #
 log.directory=Log directory
-log.directory.desc=Location of the directory for log files. The default value is ${server.output.dir}/logs.
+log.directory.desc=You can use this attribute to set a directory for all log files, excluding the console.log file, but including FFDC. By default, logDirectory is set to the LOG_DIR environment variable. The default LOG_DIR environment variable path is WLP_OUTPUT_DIR/serverName/logs. Avoid trouble: Use the LOG_DIR environment variable or com.ibm.ws.logging.log.directory property, except in cases where you are trying to change the configuration dynamically after startup.
 
 message.file.name=Message file name
-message.file.name.desc=Name of the file to which message output will be written relative to the configured log directory. The default value is messages.log.
+message.file.name.desc=Name of the file to which message output will be written relative to the configured log directory. The default value is messages.log. This file always exists, and contains INFO and other (AUDIT, WARNING, ERROR, FAILURE) messages in addition to System.out and System.err. This log also contains time stamps and the issuing thread ID. If the log file is rolled over, the names of earlier log files have the format messages_timestamp.log. Avoid trouble: Use the com.ibm.ws.logging.message.file.name property, except in cases where you are trying to change the configuration dynamically after startup.
```

Comment:
Change
"Name of the file to which message output will be written relative to the configured log directory. The default value is messages.log. This file always exists, and contains INFO and other (AUDIT, WARNING, ERROR, FAILURE) messages in addition to System.out and System.err."
to
"Name of the file to which message output is written relative to the configured log directory. The default value is messages.log. This file always exists and contains INFO and other (AUDIT, WARNING, ERROR, FAILURE) messages, in addition to System.out and System.err."